cadquery/__init__.py

```diff
@@ -1,3 +1,11 @@
+from importlib.metadata import version, PackageNotFoundError
+
+try:
+    version__ = version("cadquery")
```

Comment:
Does this work when installed via conda?

Reply:
I think it may since it leverages importlib, but it's hard to be sure. I think the `version` variable in `setup.py` is being used by conda when building packages too, so I left it hardcoded at `2.1` because I was afraid of breaking something downstream. However, we could try using the scm version number there in addition to the PyPi packages if you want, which should give us traceability down to the commit level on conda packages. I think that would be nice to be able to ask users to give the full version number when reporting bugs.

That version line has a typo in it that @fpq473 pointed out which I'll fix and I'll also give the conda version number a try just to see what happens.

Reply:
Actually, I think I'm mistaken about the conda build using the `version` variable in setup.py. There was an error in an early build that made it look that way, but I think it was actually a mistake within my setup.py changes. I think the that the cq.__version__ call should probably still work with conda, but it's hard to know for sure until we try it with the conda packages.